appium/webdriver/webelement.py

```diff
@@ -22,6 +22,49 @@
 
 
 class WebElement(SeleniumWebElement):
+    # Override
+    def get_attribute(self, name):
+        """Gets the given attribute or property of the element.
+
+        This method will first try to return the value of a property with the
+        given name. If a property with that name doesn't exist, it returns the
+        value of the attribute with the same name. If there's no attribute with
+        that name, ``None`` is returned.
+
+        Values which are considered truthy, that is equals "true" or "false",
+        are returned as booleans.  All other non-``None`` values are returned
+        as strings.  For attributes or properties which do not exist, ``None``
+        is returned.
+
+        :Args:
+            - name - Name of the attribute/property to retrieve.
+
+        Example::
+
+            # Check if the "active" CSS class is applied to an element.
+            is_active = "active" in target_element.get_attribute("class")
+
+        """
+
+        resp = self._execute(RemoteCommand.GET_ELEMENT_ATTRIBUTE, {'name': name})
+        attributeValue = resp.get('value')
+
+        if attributeValue is None:
+            return None
+
+        if not isinstance(attributeValue, str):
```

Comment:
`basestring` and `unicode` don't work in Python3. `str` is probably enough in this case. (This work both 2 and 3.)

Reply:
this coercion will throw an error in python2 if the value contains unicode chars

Reply:
http://python-future.org/compatible_idioms.html, check `basestring` and `unicode` points